dns_cert_checker.py

```diff
@@ -486,6 +529,17 @@ def main():
         help="If set, load zone information from the provided JSON file "
         "instead of requesting zone transfers from nameservers at runtime",
     )
+    parser.add_argument(
```

Comment:
This doesn't have to be addressed now, but we may want this to be stored in the config file rather than passed in as a command-line argument. The same goes for name-filters.